log4j-core/src/main/java/org/apache/logging/log4j/core/lookup/BuiltInLookupCategory.java

```diff
@@ -0,0 +1,116 @@
+package org.apache.logging.log4j.core.lookup;
+
+/**
+ * Built-in category names' home.
+ *
+ * @author tommy.testla
+ * @since 3.0.0
+ */
+public interface BuiltInLookupCategory {
```

Comment:
Two things about this:

1. We do not use interfaces for holding string constants; we use a `public final class` with `public static final` constants and a private constructor. Using interfaces this way is an anti-pattern.
2. Due to how you've refactored these to be dynamically looked up via services, these constants are only referenced in their respective classes now. There's no reason to factor out the constants at this point.

Reply:
Ok, got it.